### ISSUE TITLE
Update Installation and Run notes

### ISSUE THREAD

**Issue by SOULPWRD** [kiwicom/react-native-app#487]
I've tried to setup the project following the `Installation and Run` notes, but the thing is, I've encountered some problems regarding `ruby` and `gem` parts.

1. problem with ruby on Linux and Mac devices
Please add at least simple notes how to install `ruby` and `gem` on the host operating system using `rbenv`. For Mac users, the problem is the system version of ruby. Installing dependencies via `gem` leads to the permission [error](https://stackoverflow.com/questions/14607193/installing-gem-or-updating-rubygems-fails-with-permissions-error). The recommendation would be to install `rbenv` using homebrew and install the particular version via `rbenv` CLI.

2. Following the installation steps, with the latest stable ruby version 2.5.1, running `yarn pod-install` will end up with the failure error. The downgrade to the 2.4.4 (the second stable) version needs to be done.

I can do the pull request, plz just let me know if this issue makes sense.

**Comment by mrtnzlml** [comment]
Hi Martin!

Currently, we support only Mac devices so no need to deal with Linux whatsoever. I honestly don't remember how I installed it but I am pretty sure I didn't use `rbenv`. Therefore I am not sure if it's a good idea. What about adding it to the [Troubleshooting section](https://github.com/kiwicom/react-native-app#troubleshooting)?

Ad second point - do you know the error? Because I have the latest version and I am using it without bigger problems:

```
▲ ~ ruby --version
ruby 2.5.1p57 (2018-03-29 revision 63029) [x86_64-darwin17]
```

But I agree that it's a lot of work to make this project work. How can we improve it instead of extending our README with even more steps? This may be a good inspiration: https://github.com/artsy/emission#installation

Did you manage to run it after all?

**Comment by mrtnzlml** [comment]
ping @SOULPWRD

**Comment by SOULPWRD** [comment]
Hi Martin :)

Sorry for the late response. I tried to simulate the issue with ruby version 2.5.1, but everything worked suddenly without any problems. 😞 

Regarding `rbenv`, as I mentioned. I had the problem with ruby/gem [permissions](https://stackoverflow.com/questions/14607193/installing-gem-or-updating-rubygems-fails-with-permissions-error). When I installed ruby via `rbenv` the installation process ran smoothly. I agree with your proposal - [Troubleshooting](https://github.com/kiwicom/react-native-app#troubleshooting) section could be a good place to mention this problem/issue.

I think extending the README won't solve the technical and installation issues. I would rather split the individual sections of the README into smaller thematic groups/files which would hold the link references on each other.
Regarding the structure of the installation [notes](https://github.com/artsy/emission#installation) you posted - it looks very clean and transparent. It could be a good inspiration to use it as a template for improving the current README of this project.

Yeah, the project runs without problems. Thank you, good job 👍 🥇

**Comment by mrtnzlml** [comment]
@SOULPWRD So are you suggesting to evolve the structure of our README? The initial idea was to start with single README and then move it into separated files in `docs/` for example.

> I can do the pull request, plz just let me know if this issue makes sense.

So we will be happy if you'll help us to improve it... :)

**Comment by SOULPWRD** [comment]
Yeah sure, I can help you with the docs. 👍 I will go through the whole documentation and propose the new structure (following [the structure](https://github.com/artsy/emission#reactions--emissions) you mentioned). This could be a good starting point at least.
